hack/cmd_util.sh

```diff
@@ -400,30 +400,12 @@ function os::cmd::internal::run_until_exit_code() {
 		fi
 		return 0
 	else
-		local cause=$(os::cmd::internal::assemble_try_until_code_causes "${cmd_succeeded}")
-
-		os::text::print_red_bold "FAILURE after ${time_elapsed}s: ${description}: ${cause}"
+		os::text::print_red_bold "FAILURE after ${time_elapsed}s: ${description}: the command timed out"
```

Comment:
The timeout is embedded into the description, so we already know how long it's running:
```
FAILURE after 1.019s: hack/test-cmd_util.sh:377: executing 'exit 0' expecting failure; re-trying every 0.2s until completion or 1.000s: the command timed out
```